### ISSUE TITLE
Item names not correct

### ISSUE THREAD

**Issue by cfrankovich** [cfrankovich/ChestLogger#2]
TLDR: Item names are vague and possibly outdated.

The current state of this plugin is still in the works of actually completing its intended task. Right now, one command `/chest add` is able to get the chest the player is looking at, get the contents inside of the chest, and store all of this data on the server via a file. 

The issue lies in the part where the contents inside of the chest are needed to be recorded in order to provide the base line to the ledger feature that will be implemented later on.

The item names that these items are being "pulled" from is shown in this code snippet here
```java
Chest ch = (Chest) b.getLocation().getBlock().getState();
ItemStack[] stack = ch.getBlockInventory().getContents();
for (ItemStack s : stack)
{
        if (s == null) continue;
        Bukkit.getLogger().info(s.getType().name()); /* here */
        Bukkit.getLogger().info(Integer.toString(s.getAmount()));
}
```

When running this code with items like these...
![ss1](https://i.imgur.com/Xe93RNO.png)
This is returned in the console...
![ss2](https://i.imgur.com/nmuxbXu.png)

My suspicion of something being outdated arises with the Netherite sword and the iron ore being called "AIR." 

I've been searching online for this for a while now and haven't found anything regarding this. Is there a new(er) method to be using to retrieve item names? A bug?

**Comment by DanyPalma** [comment]
Even though you probably downloaded Bukkit/Paper/etc. as 1.18.2, you still have to define the API to whatever version you need. 

Make sure you have `api-version` inside of your `plugin.yml` defined as 1.16 or greater to have it recognize netherite items 

Ideally just have it defined as 1.18 for keeping it up-to-date for sake of efficiency 

solution found here: https://www.spigotmc.org/wiki/plugin-yml/ 

![image](https://user-images.githubusercontent.com/71732131/170835465-eada849c-9302-4472-88ed-5fa59fc2317e.png)

**Comment by cfrankovich** [comment]
Works! Thanks <3
![ss3](https://i.imgur.com/iOqTqmN.png)
